cygnus-ngsi/README.md

```diff
@@ -163,6 +163,8 @@ $ ./notification-json-simple.sh http://localhost:5050/notify myservice myservice
 
 Or you can connect a real NGSI source such as [Orion Context Broker](https://github.com/telefonicaid/fiware-orion). Please, check the [User and Programmer Guide](../doc/cygnus-ngsi/user_and_programmer_guide/connecting_orion.md) for further details.
 
+Support for NGSIv2 notifications it's added from version above 1.17.0. For this pourpose it's been added a new dir [/NGSIv2](./resources/ngsi-examples/NGSIv2) wich contans script files in order to emulate som NGSIv2 notification types. 
```

Comment:
```suggestion
Support for NGSIv2 notifications has been added from version above 1.17.1. For this purpose, it's been added a new dir [/NGSIv2](./resources/ngsi-examples/NGSIv2) which contains script files in order to emulate some NGSIv2 notification types. 
```

Reply:
Fixed in [3e19cae](https://github.com/telefonicaid/fiware-cygnus/pull/1762/commits/3e19cae79a547c62d62434cdf9e841b40ede85b3)